docs/user_guide/assessment.rst

```diff
@@ -216,6 +216,72 @@ case we see that the subgroup ``(a, 8)`` has a result of ``NaN``, indicating
 that there were no samples in it.
 
 
+.. _control_features_metrics:
+
+Control features for grouped metrics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Control features (sometimes called 'conditional' features) also divide the input
+data into subgroups. The difference is that aggregations (such as
+:code:`MetricFrame.group_max`) are performed for each subgroup in the conditional
```

Comment:
Can we make this sentence a bit more understandable please? I think the example after def. helps but if there is anyway we can make it read as, 

"Original sensitive features split the dataset into subspaces to perform fairness assessment. Conditional features make this analysis more detailed by split the original sensitive buckets (such as female and male) into more detailed sub-spaces (such as high income females and high income males) and enable a more detailed view of fairness insights.

Reply:
I've rephrased, but... control features are at a 'higher' level than sensitive features, rather than providing a finer splitting, so I've changed your proposed text a bit.